events/kbc.html

```diff
@@ -0,0 +1,158 @@
+
+    <!-- Swiper CSS -->
+    <link rel="stylesheet" href="../css/swiper.min.css">
+
+    <!-- Styles -->
+    <link rel="stylesheet" href="../style.css">
+</head>
+
+<body>
+    <header class="site-header">
+        <div class="header-bar">
+            <div class="container-fluid">
+                <div class="row align-items-center">
+                    <div class="col-10 col-lg-4">
+                        <h1 class="site-branding flex">
+                            <a href="index.html">APRATIM 2K19</a>
+                        </h1>
+                    </div>
+
+                    <div class="col-2 col-lg-8">
+                        <nav class="site-navigation">
+                            <div class="hamburger-menu d-lg-none">
+                                <span></span>
+                                <span></span>
+                                <span></span>
+                                <span></span>
+                            </div><!-- .hamburger-menu -->
+
+                            <ul>
+
+                                <li><a href="../index.html#sponsors">SPONSORS</a></li>
+                                <li><a href="../index.html#events">OUR EVENTS</a></li>
+                                <li><a href="../core.html">OUR CORE</a></li>
+                                <li><a href="../index.html#contact">CONTACT</a></li>
+                                <li><a href="../webTeam.html">WEB TEAM</a></li>
+                            </ul><!-- flex -->
+                        </nav><!-- .site-navigation -->
+                    </div><!-- .col-12 -->
+                </div><!-- .row -->
+            </div><!-- container-fluid -->
+        </div><!-- header-bar -->
+    </header><!-- .site-header -->
+
+    <div class="hero-content">
+        <div class="container">
+            <div class="row">
+                <div class="col-md-8 offset-md-2 text-center">
+                    <img src="../images/cover.jpg" alt="logo">
+                </div>
+            </div>
+            <div class="row">
+                <div class="col-md-10 offset-md-1 text-center entry-header">
+                    <h2>KBC Quiz</h2>
+                </div><!-- .col-md-10 -->
+            </div>
+        </div><!-- .container -->
+    </div><!-- .hero-content -->
+
+    <div class="content-section2">
+        <div class="container">
+            <div class="row">
+
+               <div class="col-md-6 lineup-artists">
+                    <figure class="featured-image">
+                        <img src="../images/kbc.jpg" alt="">
+                                    <!-- <a href="#" class="box-link"> <img src="images/box.jpg" alt=""> </a> -->
+                    </figure>                   	
+                </div>
+                <div class="col-md-6 lineup-artists-headline">
+                    <h2 class="entry-title">Rules</h2>
+                    <ol>
+                        <li><p> Maximum team size is of 2 students.</p></li>
```

Comment:
There is no need to add a `<p></p>` inside the `<li></li>` tags.
Please make the required changes!